Sources/Brave/Frontend/Browser/Playlist/Controllers/PlaylistNewFolderView.swift

```diff
@@ -41,7 +41,7 @@ class PlaylistFolderImageLoader: ObservableObject {
   func load(domainUrl: URL) {
     faviconTask?.cancel()
     faviconTask = Task { @MainActor in
-      let favicon = try await FaviconFetcher.loadIcon(url: domainUrl, persistent: !PrivateBrowsingManager.shared.isPrivateBrowsing)
+      let favicon = try await FaviconFetcher.loadIcon(url: domainUrl, persistent: true)
```

Comment:
Yeah correct. Playlist doesn't really know anything about Private-Browsing since it doesn't have any different functionality depending on the mode. I just found it weird that adding videos in private-mode wouldn't saved-offline icon because private-browsing doesn't save to disk the icons.